charms/kfp-metadata-writer/requirements.in

```diff
@@ -4,4 +4,7 @@
 charmed-kubeflow-chisme >= 0.2.0
 lightkube
 ops
+# pydantic greater or equal to 2.7 yields issues with the
+# rustc version available for this ubuntu version
+pydantic==2.6.4
```

Comment:
Not a bug exactly, but I noticed during testing that build times have increased by ~8min:
* [before](https://github.com/canonical/envoy-operator/actions/runs/8569177476):
  * publish: ~3-4min
  * integration: 14min
* [after](https://github.com/canonical/envoy-operator/actions/runs/8724143722): 
  * publish: ~11min
  * integration: 21min
  
I'm 99% certain this is because of the time taken to build pydantic.  I'm documenting this in the mlops repo

Reply:
Yes, that's the reason. Thanks for filing the issue, we can discuss this subject there. I don't think there is an action item in this PR atm, but let me know otherwise.